plugin.xml

```diff
@@ -44,6 +44,22 @@
 
     <framework src="com.google.android.gms:play-services-auth:10.2+"/>
 
+    <!-- Configuration of OpenID auth plugin start -->
+    <framework src="net.openid:appauth:0.7.0"/>
+    <config-file target="AndroidManifest.xml" parent="/manifest/application">
+      <activity android:name="net.openid.appauth.RedirectUriReceiverActivity">
+        <intent-filter>
+          <action android:name="android.intent.action.VIEW"/>
+          <category android:name="android.intent.category.DEFAULT"/>
+          <category android:name="android.intent.category.BROWSABLE"/>
+          <data android:scheme="emission.auth" />
```

Comment:
when you said that you needed a new URL scheme, this is exactly what I thought you would do.
So cool!